app/kubernetes-ingress-controller/1.0.x/concepts/ingress-versions.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Ingress v1 and v1beta1 differences
+---
+
+## Introduction
+
+Kubernetes 1.18 introduced a new `networking.k8s.io/v1` API for the [Ingress resource][kubernetes-ingress-doc].
+It standardizes common practices and clarifies implementation requirements that
+were previously up to individual controller vendors. This document covers those
+changes as they relate to Kong Ingress Controller and provides example
+equivalent `networking.k8s.io/v1beta1` and `networking.k8s.io/v1` resources for comparison.
+
+## Paths
+
+Both Ingress v1beta1 and v1 HTTP rules require a path, which represents a [URI
+path][uri-rfc-paths]. Although v1beta1 had specified that paths were [POSIX
+regular expressions][posix-regex] and enforced this, but in practice most
+controllers used other other implementations that did not match the
+specification. v1 seeks to reduce confusion by introducing several i[path
+types][path-types] and lifting restrictions on regular expression grammars used
+by controllers.
+
+### `networking.k8s.io/v1beta1`
+
+The controller passes paths directly to Kong and relies on its [path handling
+logic][kong-path]. The Kong proxy treats paths as a prefix unless they include
+characters [not allowed in RFC 3986 paths][uri-rfc-paths], in which case the
+proxy assumes they are a regular expression, and does not treat slashes as
+special characters. For example, the prefix `/foo` can match any of the
+following:
+
+```
+/foo
+/foo/
+/foobar
+/foo/bar
+```
+
+### `networking.k8s.io/v1`
+
+Although v1 Ingresses provide path types with more clearly-defined logic, the
+controller must still create Kong routes and work within the Kong proxy's
+routing logic. As such, the controller translates Ingress rule paths to create
+Kong routes that match the specification:
```

Comment:
```suggestion
Kong routes that match one of the following specifications: `Exact`, `Prefix`, or `ImplementationSpecific`.
```
Mostly because it's not great to leave a `:` right before a header.